src/app/schedule-view/component.ts

```diff
@@ -141,4 +140,10 @@ export class ScheduleViewComponent implements OnInit, OnDestroy {
     }
     return this.schedules[this.scheduleIndex].statusText;
   }
+
+  public clear (): void {
+    for (let i in this.courses) {
```

Comment:
This would work most of the time, but could be problematic in the future if we change this class. Instead of looping over the courses here, can we add a method to the `SelectionsService` to erase all of the sections, and call that here?